ansys_optical_automation/application/Run_Speos_Labs_Admin.py

```diff
@@ -0,0 +1,151 @@
+"""
+Requirements: administrative privileges.
+-----------------------------------------------------------------------
+This module launches with administrator privileges all executables in:
+`C:\\Program Files\\ANSYS Inc\\vXXX\\Optical Products\\Viewers`
+
+Many users lack administrative rights, so this module can be executed right after the installation
+while the user still has admin rights.
+
+Upon launching each application, the module registers the associated COM server used to call each application, 
+with the path to each executable appropriately registered.
+
+Classes and Main Methods
+-----------------------------------------------------------------------
+1. **SpeosPathFinder**
+   - This class provides two methods to locate the Speos application path:
+      - A method to search for a specific Speos version Path.
+      - A default method that identifies and returns the latest installed version of Speos.
+
+2. **LabsAdmin**
+   - This class offers methods to open or close one or multiple applications simultaneously.
+   - By default, if no specific applications are specified, it will launch all applications found.
+
+"""
+
+import subprocess, os, time, sys
+
+# If pygetwindow is not installed, do it
+third_party_package = "pygetwindow"
+try: 
+    gw =  __import__(third_party_package)
+    print(f"{third_party_package} is already installed.")
+except ImportError:
+    print(f"{third_party_package} not found. Installing...")
+    subprocess.check_call([sys.executable, "-m", "pip", "install", third_party_package])
+    import pygetwindow as gw
+
+class SpeosPathFinder:
```

Comment:
function already available in the sdcm_core/utility.py: [link](https://github.com/ansys/optical-automation/blob/c78b3377c348424d35c29e75ba9da4323fdea582/ansys_optical_automation/scdm_core/utils.py#L8)
please consider using / improving that function.

Reply:
@pluAtAnsys updated, can you check next commit ?

I created a next function that is called from get_scdm_install_location, as in that function you are looking for scdm folder.